### ISSUE TITLE
Arkane Error with Source Install using "Bash on Ubuntu on Windows 10"

### ISSUE THREAD

**Issue by cjmcgill** [ReactionMechanismGenerator/RMG-Py#1516]
### Bug Description
Main branch install for Arkane onto Windows from source doesn't work anymore per issue #1507 . I was looking for a way that I could get an updated source install onto my Windows machine, so I installed into the "Bash on Ubuntu on Windows 10" subenvironment. The install itself and updates from git seem to go through fine.

Arkane seems to run the thermo function without error. The output files include a fitted NASA polynomial and other parameters as expected.

There is an error when I run Arkane with an input using the kinetics function. Interestingly, the output file generated still includes the rate constants for the first reaction called for, but in an input file containing multiple reactions it stops at the first.

Since the failure doesn't occur for the thermo function and seems to get through most of the kinetics function, I am hoping that there's a fix possible here. Maybe even something I can do like setting up an environmental variable or something.

```
  File "../RMG-Py/arkane.py", line 60, in <module>
    arkane.execute()
  File "/home/cjmcgill/RMG/RMG-Py/arkane/main.py", line 298, in execute
    job.execute(outputFile=outputFile, plot=self.plot)
  File "/home/cjmcgill/RMG/RMG-Py/arkane/kinetics.py", line 140, in execute
    self.draw(os.path.dirname(outputFile))
  File "/home/cjmcgill/RMG/RMG-Py/arkane/kinetics.py", line 362, in draw
    KineticsDrawer().draw(self.reaction, format=format, path=path)
  File "/home/cjmcgill/RMG/RMG-Py/arkane/kinetics.py", line 516, in draw
    import cairocffi as cairo
  File "/home/cjmcgill/anaconda2/envs/rmg_env/lib/python2.7/site-packages/cairocffi/__init__.py", line 40, in <module>
    'cairo', 'libcairo-2')
  File "/home/cjmcgill/anaconda2/envs/rmg_env/lib/python2.7/site-packages/cairocffi/__init__.py", line 34, in dlopen
    return ffi.dlopen(names[0])  # pragma: no cover
  File "/home/cjmcgill/anaconda2/envs/rmg_env/lib/python2.7/site-packages/cffi/api.py", line 141, in dlopen
    lib, function_cache = _make_ffi_library(self, name, flags)
  File "/home/cjmcgill/anaconda2/envs/rmg_env/lib/python2.7/site-packages/cffi/api.py", line 802, in _make_ffi_library
    backendlib = _load_backend_lib(backend, libname, flags)
  File "/home/cjmcgill/anaconda2/envs/rmg_env/lib/python2.7/site-packages/cffi/api.py", line 798, in _load_backend_lib
    return backend.load_library(path, flags)
OSError: cannot load library '/home/cjmcgill/anaconda2/envs/rmg_env/lib/libcairo.so.2': libXrender.so.1: cannot open shared object file: No such file or directory
```
The file libcairo.so.2 is located at the filepath location listed in the error message. I'm not sure where libXrender.so.1 is located, it doesn't seem to be in that same directory.

Attaching folder of input and output files from a test of the error.
[Results.zip](https://github.com/ReactionMechanismGenerator/RMG-Py/files/2632164/Results.zip)

### How To Reproduce
Install per the linux source install instructions with anaconda. Activate rmg_env and run an arkane file with a kinetics command.

### Expected Behavior
Generate arkane output files for kinetics calculation.

### Installation Information
Describe your installation method and system information.
 - OS: Windows 10 in "Bash on Ubuntu on Windows 10" subenvironment
 - Installation method: linux installation from source with anaconda
 - RMG version information:
   - RMG-Py: 2.1.0-1519-gae79625

**Comment by cjmcgill** [comment]
I had been thinking this was a RMG specific file, but googling around it actually looks like xrender1 is an available library out there. I installed it and Arkane seems to work as expected now for the kinetics.

`sudo apt-get install libxrender1`

I'm not closing out the thread because there's still the question of why the library wasn't installed with the rest of it like it presumably should have. Is that a default library in most linux environments that just missing in this subenvironment or something?

**Comment by alongd** [comment]
Hi @cjmcgill,
In my understanding, the missing package as suggested by the error trace, cairolib, is known to be problematic on Windows. I'm unfamiliar with `libxrender1`. Reading online I see it's a tailored library for the X Windows system. Is it possible that "Bash on Ubuntu on Windows 10" identified it as a substitution? I'm glad to hear that Arkane now works for you in this configuration. Were there any additional change you had to make to run it on "Bash on Ubuntu on Windows 10"?

**Comment by cjmcgill** [comment]
As well as I can tell, installing libxrender1 is the only thing out of the normal installation procedure that I did. I’ve only run thermo and high-pressure-limit calculations in Arkane. Using other RMG/Arkane functions could turn up other gaps.

**Comment by amarkpayne** [comment]
Unrelated to this, I just tried installing RMG on the Ubuntu subsystem on Windows 10 as part of an effort to update our Windows installation documentation. I ran into the following error when trying to run `eg0`. I also tried installing libXrender1 with 

`sudo apt-get install libxrender1`

and it solved my problem as well. I then saw this post that encountered the same issue.

Given that this error seems somewhat common, I'll add this to the installation instructions that I am making so that future users install this library if they choose the Ubuntu subsystem route. Thanks @cjmcgill for pointing this issue out as well!

```
==============================
*** Open Babel Error  in openLib
 /home/ampayne/anaconda2/envs/rmg_env/lib/openbabel/2.4.1/xmlformat.so did not load properly.
Error: libXrender.so.1: cannot open shared object file: No such file or directory
==============================
```
